crates/refunder/Cargo.toml

```diff
@@ -14,6 +14,7 @@ database = { path = "../database" }
 ethcontract = { workspace = true }
 futures = {workspace = true}
 lazy_static = { workspace = true }
+model = { path = "../model" }
```

Comment:
It's a bit weird that the `refunder` now has to depend on the `model` crate (which is for the API JSON structs AFAIU) to get a timestamp.
I think eventually having a crate with a bunch of small helper utilities like this (with no or very few dependencies) could make sense. But this probably has to be thought through a bit more.